PyInstaller/utils/hooks/__init__.py

```diff
@@ -1,35 +1,23 @@
-#-----------------------------------------------------------------------------
+# ----------------------------------------------------------------------------
```

Comment:
Please do not add spaces here. None of our copy-right-headers has them. I know, this i s nit.picking at it's best. But for the case we need to change the header (and will do that automatically), these will fall on our feed.